src/main/java/io/supertokens/webserver/WebserverAPI.java

```diff
@@ -225,7 +233,26 @@ private String getConnectionUriDomain(HttpServletRequest req) {
     }
 
     protected TenantIdentifier getTenantIdentifier(HttpServletRequest req) {
-        return new TenantIdentifier(this.getConnectionUriDomain(req), this.getAppId(req), this.getTenantId(req));
```

Comment:
there used to be some part in the code wherein it used to call the resource distributor to get the config of the tenant to check if the tenant exists and refresh the tenant list if it didn't exist in this core's memory - where did that go?